### ISSUE TITLE
3D OBJ is missing

### ISSUE THREAD

**Issue by skydiveguy** [smeighan/xLights#2126]
**Describe the bug**
OBJ file is selected but not showing up in MacOS Layout file

**To Reproduce**
1. Go to Layout tab
2. Click on 3D Objects
3. ObjFile is listed and exists and Active is selected but it is not displayed

**Expected behavior**
It should be showing the obj mesh of the house layout
![objmissing](https://user-images.githubusercontent.com/67765665/86993343-c3e12980-c171-11ea-9fb2-c7b4b527d70f.png)

**Versions (please complete the following information):**
 - OS: MacOS Catalina
 - xLights version 2020.27

**Comment by keithsw1111** [comment]
Going to need more ... maybe tools ... package logs. And create a new empty sequence and package it so we get all the files.

**Comment by dkulp** [comment]
As noted in the README/release notes, the Mac builds are now sandboxed.  As such, any resources/files outside of your show folder will NEED to be re-selected so that the Apple sandbox security mechanisms will know that you explicitly gave xLights permission to read those files.

**Comment by skydiveguy** [comment]
I read the README and went into the specified "Preferences" area as specified and checked all settings. I have a pretty basic setup and everything looked fine.
I also did reselect the obj file and save in the Layout/3D section but it is not having any effect.

**Comment by skydiveguy** [comment]
> Going to need more ... maybe tools ... package logs. And create a new empty sequence and package it so we get all the files.

[xLightsProblem.zip](https://github.com/smeighan/xLights/files/4896400/xLightsProblem.zip)
[BugReportBlank.zip](https://github.com/smeighan/xLights/files/4896410/BugReportBlank.zip)

**Comment by AzGilrock** [comment]
If you look at the log file in the xLightsProblem.zip file you provided it tells you the error.  Imagine that its exactly what Dan said so please read the developer suggestions.

WARN: Mesh object 'Mesh' obj file '/Users/brian/Google Drive/xLights/3D/590Exterior.obj/590Exterior.obj' not under show directory.

**Comment by skydiveguy** [comment]
Im not a developer so I dont completely understand what Id be looking for in the log files (which is why Im submitting the problem here).
As for your solution... as I know this is an evolving version with sandboxing (as listed in the release notes to expect some issues, which is why im trying to make the devs aware of potential items that other might report) will there be a way to allow previously used locations (outside the show folder) to work? I have an absolute file path listed there so I dont see why it cant navigate to that location to get the file its looking for or at least a setting to allow xlights to access locations outside of the show folder (again, Im not really sure how it works as Im not a dev and perhaps its a limitation of the software).
Not trying to sound like Im arguing/complaining, Im just trying to point out issues that others might potentially have and trying to save you from dealing with them.
Thank you for the fast response.

**Comment by AzGilrock** [comment]
I personally like to use a subdirectory under my show folder to hold all the assets needed for a sequence.  That's the safest method to make sure you can copy that folder for next year and not mess things up.  Just use that method and be happy.

**Comment by skydiveguy** [comment]
I will move my files and folders as requested. 
Last year was my first year and I didnt fully understand the implications of the folder structure at the time and I rebuilt my layout this year with everything (I thought) under one "show" folder. For some reason when I moved the sequences over I didnt move any "image" files and didnt think about it becasue it "just worked".
Just be aware that I may not be the only one with this issue and it will probably need to be addressed most likely with documentation on having to store everything under the show folder going forward.
Thanks again.
